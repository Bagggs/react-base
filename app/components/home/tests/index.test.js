import React from 'react';
import renderer from 'react-test-renderer';
import Home from 'components/home';

describe('Home', () => {
  const homeComponent = renderer.create(<Home/>).toJSON();

  it('renders correctly', () => {
    expect(homeComponent).toMatchSnapshot();
  });
});
