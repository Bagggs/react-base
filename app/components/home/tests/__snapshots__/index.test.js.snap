// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="jumbotron"
  >
    <h1>
      React-base
    </h1>
    <p>
      Kick-start your new web application based on React and Redux technologies.
      <br />
      It also includes Webpack, React hot loader, PostCSS, JSON-server tools for even more rapid development.
    </p>
  </div>
</div>
`;
